surfsense_backend/app/tasks/background_tasks.py

```
@@ -43,20 +45,21 @@ async def add_crawled_url_document(
         if type(crawl_loader) == FireCrawlLoader:
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)